xyz.safewords.midiconn.yml

```
@@ -0,0 +1,63 @@
+command: midiconn
+id: xyz.safeworlds.midiconn
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+modules:
+  - name: SDL2
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/libsdl-org/SDL
+        tag: release-2.26.1
+  - name: fmt
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DFMT_DOC=OFF
+      - -DFMT_INSTALL=ON
+      - -DFMT_TEST=OFF 
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://github.com/fmtlib/fmt
+        tag: '8.1.1'
+  - name: spdlog
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://github.com/gabime/spdlog
+        tag: v1.10.0
+  - name: rtmidi
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DRTMIDI_API_JACK=OFF
+      - -DRTMIDI_BUILD_STATIC_LIBS=ON
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://github.com/thestk/rtmidi
+        commit: 806e18f575b68c23b26f9398e1b6866b335b5308
+  - name: midiconn
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://gitlab.com/mfep/midiconn.git
+        tag: 0.1.0
+cleanup:
+  - /bin/sdl2-config
+  - /include
+  - /lib/cmake
+  - /lib/debug
+  - /lib/pkgconfig
+  - /lib/*.a
+  - /lib/*.la
+  - /share/aclocal
+  - /share/rtmidi
+finish-args:
+- --socket=x11
```

```suggestion
- --share=ipc
```

this is needed for x11.

---

I'm not sure if I understand correctly. The current set of `finish-args` works both in a Wayland and X session, at least on my system. The proposed one doesn't. Can you please elaborate?

---

OK, probably you meant to add `--share=ipc` additionally, as the CI job also suggests. Done.